src/scripts/mapper/Initialisierung.lua

```diff
@@ -22,7 +22,7 @@ end)
 enableMapInfo("Kartenmodus")
 
 function echoM(str)
```

Comment:
Wahrscheinlich sollten wir die Funktionen auch in irgendeinem Table gruppieren.. 
Aber lieber in einem seperaten PR. Der hier wird schon groß genug 😄